appveyor.yml

```diff
@@ -1,30 +1,105 @@
-platform:
-  - x86
-  - x64
+platform: x64
```

Comment:
is it ok to compile 32bit applications on a 64bit platform (like we do on Travis)?
Setup is a lot easier this way